sdk/identity/azure-identity/src/client_certificate_credential.cpp

```diff
@@ -50,6 +60,204 @@ template <typename T> std::vector<uint8_t> ToUInt8Vector(T const& in)
   return outVec;
 }
 
+using CertificateThumbprint = std::vector<unsigned char>;
+using UniquePrivateKey = Azure::Identity::_detail::UniquePrivateKey;
+using PrivateKey = decltype(std::declval<UniquePrivateKey>().get());
+
+#ifdef AZ_PLATFORM_WINDOWS
```

Comment:
Not a huge deal, but I tend to prefer `#if defined(XXX)` over `#ifdef XXX` because `if defined` can be added to unlike `#ifdef`.

Reply:
Done.